### PR TITLE
Python 3 compatibility

### PR DESCRIPTION
**Pull request by rouge8**
I added Python 3 compatibility and updated the Circle CI config to use tox to test 2.7, 3.6, and 3.7.

### CODE REVIEW

**Comment by rouge8** [comment]
@pior It looks like Circle CI isn't triggering, and I haven't used it before, so no promises about the Circle CI config...

**Comment by pior** [comment]
Thanks @rouge8 !
I will take a look at this tonight, the CI should run.

**Comment by pior** [comment]
The option to build from **forks** was OFF in the CircleCi config. 
I enabled it. 

Could you try pushing in your PR to test it?

File: pyramid_useragent/tests.py [hunk 1]
```diff
@@ -48,7 +49,7 @@ def test_value_none(self):
 
         ua = UserAgent(None)
 
-        self.assertItemsEqual(ua.components, [])
+        six.assertCountEqual(self, ua.components, [])
```
Comment (pior) [inline pyramid_useragent/tests.py:52]: TIL 👌

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,32 +1,47 @@
-version: 2
-jobs:
-  build:
-    docker:
-      - image: circleci/python:2.7.14
+version: 2.1
```
Comment (pior) [inline .circleci/config.yml:1]: So CircleCI supports tox natively now, amazing.

**Review by pior — APPROVED** [review]
We will merge this as soon as we confirm it runs in CI.
Thanks!

**Comment by rouge8** [comment]
Oh boy, getting Circle CI working will take some trial and error 😩

**Comment by rouge8** [comment]
Aha!

@pior in order to use the CircleCI 2.1 config, you need to update the project config:

> Configuration version 2.1 requires the "Enable Build Processing" project setting. Enable Build Processing under Project Settings -> Advanced Settings.

I used `circleci config process` to convert the 2.1 config to version 2 and can revert that once you update the project setting 😌

**Comment by pior** [comment]
@rouge8 I enabled 2.1 👍

**Comment by rouge8** [comment]
@pior it worked!

**Comment by pior** [comment]
Love it! 
I will push a release tonight.

**Comment by rouge8** [comment]
Thanks! 🎉

**Comment by pior** [comment]
Just released 0.4.0

**Pull request merged**